front/src/components/boxs/device-in-room/DevicesBox.jsx

```diff
@@ -79,11 +118,24 @@ class DevicesComponent extends Component {
     });
   };
 
+  changeAllLightsStatusRoom = async () => {
+    const newValue = this.getLightStatus() === 0 ? 1 : 0;
+    // Foreach device features
+    await Promise.map(this.state.deviceFeatures, async feature => {
+      const isLightBinary =
+        feature.category === DEVICE_FEATURE_CATEGORIES.LIGHT && feature.type === DEVICE_FEATURE_TYPES.LIGHT.BINARY;
+      // if device feature is a light, we control it
+      if (isLightBinary) {
+        return this.updateValue(null, null, null, feature, null, null, newValue);
+      }
+    });
+  };
+
   //Remove x, y when DeviceInRoom is rewrite without action
```

Comment:
is it possible now to remove x, y arguments ?

Reply:
done!